components/gitpod-db/src/typeorm/migration/1704350022475-TrackEmailForFreeCredits.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists, indexExists } from "./helper/helper";
+
+export class TrackEmailForFreeCredits1704350022475 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const table = "d_b_free_credits";
+        const column = "email";
```

Comment:
Should we redact email?

Reply:
I don't think so. This is for abuse prevention, so it's fine to track it over here. :+1:

Reply:
OK

(asking because I thought delete accound would clean all sensitive / plain text that relates to the user, includes email addr)